.vscode/settings.json

```
@@ -23,5 +23,20 @@
     "sonarsource",
     "swcrc",
     "virtuals"
-  ]
+  ],
+  "editor.defaultFormatter": "biomejs.biome",
+  "[typescript]": {
+    "editor.defaultFormatter": "biomejs.biome"
+  },
+  "[javascript]": {
+    "editor.defaultFormatter": "biomejs.biome"
+  },
+  "[json]": {
+    "editor.defaultFormatter": "biomejs.biome"
+  },
+  "[jsonc]": {
+    "editor.defaultFormatter": "biomejs.biome"
+  },
+  "typescript.validate.enable": false,
+  "javascript.validate.enable": false
```

This line is enough, remove everything else

```suggestion
  "editor.defaultFormatter": "biomejs.biome",
```